pages/common/dict.md

```diff
@@ -0,0 +1,24 @@
+# dict
+
+> Command line dictionary, using DICT protocol.
```

Comment:
```suggestion
> Command line dictionary using the DICT protocol.
```

Reply:
Yay, finally XD, thank you so much @marchersimon.